lib/axon/updates.ex

```diff
@@ -163,7 +177,21 @@ defmodule Axon.Updates do
       * `:eps` - numerical stability term. Defaults to `1.0e-7`
 
   """
-  def scale_by_rss(combinator \\ identity(), opts) do
+  def scale_by_rss() do
+    scale_by_rss(identity(), [])
+  end
+
+  def scale_by_rss(opts) when is_list(opts) do
```

Comment:
```suggestion
  def scale_by_rss(combinator_or_opts \\ [])

  def scale_by_rss(opts) when is_list(opts) do
```

You can apply similar technique everywhere. :) It will save you 2 LOC per function :P